src/EnergyPlus/DataOutputs.hh

```diff
@@ -96,13 +96,13 @@ struct OutputsData : BaseGlobalStruct
 
     int MaxConsideredOutputVariables = 0; // Max Array size for OutputVariable pre-scanned
     int NumConsideredOutputVariables = 0; // Number of variables - pre-scanned, allowed for output
-    int iNumberOfRecords;                 // Number of records in input
-    int iNumberOfDefaultedFields;         // number of defaulted fields
-    int iTotalFieldsWithDefaults;         // number of fields that can be defaulted
-    int iNumberOfAutoSizedFields;         // number of autosized fields
-    int iTotalAutoSizableFields;          // number of fields that can be autosized
-    int iNumberOfAutoCalcedFields;        // number of autocalculated fields
-    int iTotalAutoCalculatableFields;     // number of fields that can be autocalculated
+    int iNumberOfRecords = 0;             // Number of records in input
```

Comment:
The warning list in Visual Studio identified a bunch of other uninitialized variables. So, why not clean up all of them (or most of them)?

Reply:
Yeah, shouldn't cause scope creep for your time, right? :upside_down_face: